opentok/opentok.py

```diff
@@ -535,6 +549,9 @@ def start_archive(
             "resolution": resolution,
         }
 
+        if layout is not None:
```

Comment:
I'm assuming this is just meant to be passthrough and the user is meant to build the `layout` object ahead of time? Guess we just never allowed folks to start an archive with a layout before in this SDK?